### PR TITLE
Review: 13-spatial-cv

### PR DESCRIPTION
**Pull request by pat-s**
@jannes-m 

Discussion open!

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Many thanks for this Patrick - looks like a useful set of comments and suggestions I imagine will be merged next week, when I'll be in Jena + Nuenburg!

File: 13-spatial-cv.Rmd [hunk 1]
```diff
@@ -168,15 +168,16 @@ tm_shape(hs, bbox = bbx) +
 
 
 ## Conventional modeling approach in R {#conventional-model}
-Later on we will introduce the **mlr** package, an umbrella-package providing a unified interface to a plethora of modeling approaches.
```
Comment (jannes-m) [inline 13-spatial-cv.Rmd]: algorithms is a bit too generic I'd say

File: 13-spatial-cv.Rmd [hunk 2]
```diff
@@ -168,15 +168,16 @@ tm_shape(hs, bbox = bbx) +
...
 This way we introduce statistical supervised modeling in R which provides the required skill set for doing spatial CV and additionally contributes to a better grasp on the **mlr** approach introduced later on.
 Usually, we model the response variable as a function of predictors. 
-Therefore, modeling functions in R such as `lm`, `glm` and many more use the so-called formula interface.
+Therefore, most algorithms implementations in R such as `lm`, `glm` and many more use the so-called formula interface.
 We put this into practice by modeling the landslide occurrence as a function of terrain attributes.
-Since our response (landslide occurrence) belongs to the binary category, we use a binomial generalized linear model instead of a simple linear model which would expect a normally distributed numeric response variable.
+Since our response (landslide occurrence) is binary, we use a generalized linear model with a binomial family and a logit link function instead of a simple linear model which does not allow for other response distributions besides Gaussian.
 
 ```{r, eval = TRUE}
+# meist wird einfach "binomial" geschrieben anstatt binomial()
```
Comment (jannes-m) [inline 13-spatial-cv.Rmd]: yep, this is right but this is what the mlr package is using internally, so I adopted this here (just to show that mlr is doing exactly the same)

File: 13-spatial-cv.Rmd [hunk 3]
```diff
@@ -204,6 +205,8 @@ pred = raster::predict(object = ta, model = fit,
                        type = "response")
 ```
 
+Ich würde maps immer lat/lon darstellen, UTM coords find ich immer undescriptive.
```
Comment (jannes-m) [inline 13-spatial-cv.Rmd]: na ja, da sind wir wohl unterschiedlicher Auffassung -:)

File: 13-spatial-cv.Rmd [hunk 4]
```diff
@@ -226,15 +229,18 @@ tm_shape(hs, bbox = bbx) +
 
 ```
 
-Here, when making predictions we neglect spatial autocorrelation since we assume that on average predictive accuracies of parametric models remain the same with or without spatial autocorrelation structures.
+Here, when making predictions we neglect spatial autocorrelation since we assume that on average the predictive accuracy remains the same with or without spatial autocorrelation structures.
+# Sp. autocor. structures can afaik only included during model fitting?
```
Comment (jannes-m) [inline 13-spatial-cv.Rmd]: universal kriging constructs a model and then estimates the spatial autocorrelation from the residuals. Finally, the interpolated residuals are added/subtracted from the predictions.

File: 13-spatial-cv.Rmd [hunk 5]
```diff
@@ -266,20 +272,24 @@ For example, a 100-repeated 5-fold cross-validation means to randomly split the
 This guarantees that each observation is used once in one of the test sets, and requires the fitting of five models.
 Subsequently, this procedure is repeated 100 times.
 Of course, the data splitting will differ in each repetition.
-Overall, this amounts to fitting 500 models whereas the mean performance measure (AUROC) of all models is the model's overall prediction power.
+# if the error is calc. on the fold-level. most often its calc. on the repetition level. maybe worth noting.
```
Comment (jannes-m) [inline 13-spatial-cv.Rmd]: let's talk about this in person

File: 13-spatial-cv.Rmd [hunk 6]
```diff
@@ -266,20 +272,24 @@ For example, a 100-repeated 5-fold cross-validation means to randomly split the
...
-Using this information in our modeling is like a sneak preview, i.e. using information that should be unavailable to the training dataset.
-To alleviate this problem, we should make use of spatial partitioning which splits the observations into spatially disjoint folds (using the observations' coordinates in a *k*-means clustering; @brenning_spatial_2012; second row of Figure \@ref(fig:partitioning)).
+This means these points are not statistically independent because training and test points in conventional cross-validation are often too close to each other (see first row of \@ref(fig:partitioning)).
+# this should be rephrased, maybe: Using this information during model fitting is like cheating in a test, i.e. testing the model on observations that are almost identical to the ones it was trained on.
+Using this information during model fitting is like a sneak preview, i.e. using information that should be unavailable to the training dataset.
+# "folds" only for the repetition split, "partitions" or "subsets" for splitting within a fold
```
Comment (jannes-m) [inline 13-spatial-cv.Rmd]: ok, another thing to talk about

**Comment by jannes-m** [comment]
@pat-s Thanks so much for your valuable comments and input! Let us talk in person about the remaining issues.
